### DISCUSSION TITLE
How to disable error message for offline mode

### DISCUSSION THREAD

**Discussion by yuriydrobniy** [rnmapbox/maps#3347] [Q&A]
How to disable error message like this for offline mode: ExceptionsManager.js:158 Mapbox error Map load failed: {tile-id={x=22222, y=33333, z=19}, type=tile, message=Failed to load tile: Unable to resolve host "api.mapbox.com": No address associated with hostname, begin=22349538336, source-id=mapbox://mapbox.satellite}?

**Comment by mfazekas** [comment]
See https://github.com/rnmapbox/maps/blob/main/docs/Logger.md#setlogcallbacklogcallback
